libs/react/src/lib/link/link.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { HTMLProps } from 'react'
+
+interface LinkProps {
+  link?: string
+  text?: string
+}
+
+export const Link = ({ link, text }: LinkProps) => {
+  const props: HTMLProps<HTMLAnchorElement> = {}
+  if (link) props.href = link
+
+  return (
+    <div>
```

Comment:
what is this div for?

Reply:
No purpose, we just overlooked it. We will remove it in the next commit.